packages/services/f-analytics/src/services/_tests/analytics.service.test.js

```diff
@@ -409,19 +407,17 @@ describe('Analytic Service ::', () => {
             const expected = {
                 ...defaultState.pageData,
                 group: 'test-feature-name',
-                name: 'test-page-name',
                 conversationId: 'f0740341-4369-437d-bcce-735a71ee5b78',
-                requestId: undefined,
-                orientation: 'Landscape',
-                display: 'mid'
+                display: 'mid',
```

Comment:
And again here; are the `display` and `orientation` values relevant to this test?

Reply:
Maybe not but as we use the real helper to calc these values are we mock the display above we have to provide valid values and not rely on the default.